app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php

```diff
@@ -447,8 +445,8 @@ public function getAllPolygonsAsGeoJSONDownload(Request $request)
             foreach ($polygonsUuids as $polygonUuid) {
                 $feature = [];
                 $polygonGeometry = PolygonGeometry::where('uuid', $polygonUuid)
-                  ->select(DB::raw('ST_AsGeoJSON(geom) AS geojsonGeom'))
-                  ->first();
+                ->select(DB::raw('ST_AsGeoJSON(geom) AS geojsonGeom'))
+                ->first();
```

Comment:
Please bring the indentation back on these two lines and 391/392

Reply:
done